<ui version="4.0" >
 <class>CaptionDialog</class>
 <widget class="QWidget" name="CaptionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>captionDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="KFontChooser" name="m_commentsFontChooser" />
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Colors</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Font color:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>55</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" >
         <widget class="KColorButton" name="m_commentsFontColor" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Background color:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>55</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2" >
         <widget class="KColorButton" name="m_commentsBgColor" />
        </item>
        <item row="2" column="0" colspan="2" >
         <widget class="QCheckBox" name="m_transparentBgCheckBox" >
          <property name="text" >
           <string>Transparent background</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Line length (in characters)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="m_commentsLinesLengthSpinBox" >
           <property name="buttonSymbols" >
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>500</number>
           </property>
           <property name="value" >
            <number>72</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontChooser</class>
   <extends>QWidget</extends>
   <header>kfontdialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
